sieve_of_eratosthenes/sieveOfEratosthenes.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include <string.h>
+#include <math.h>     // for using square root 
+#include <stdbool.h>  // to include bool datatype 
+
+void sieveOfEratosthenes(int n) {
+	bool primes[n+1];
+	int i, sqrt_n = sqrt(n)+1, j;
```

Comment:
@Monal5031: I don't think that that's the case with C. If you are inferring this from the revert PR I created, that was for Java.